util/schema/registry_schema.json

```diff
@@ -295,7 +292,6 @@
 						},
 						"label": {
 							"enum": [
-								"CC-BY",
 								"CC BY 2.0",
 								"CC-BY 2.0"
```

Comment:
Shouldnt we just delete this too? the `CC-` notation with `-`?

Reply:
Sure, but it's not clear whether we should use the Creative Commons way of writing it (no dashes) or the SPDX way with dashes https://spdx.org/licenses/CC-BY-2.0.html

I think it would be nice to say this does indeed defer to a controlled vocabulary

Reply:
Okay so the solution was to use the official CC abbreviations and not allow anything else